src/config/index.js

```diff
@@ -4,26 +4,83 @@ const path = require('path');
 
 const fse = require('fs-extra');
 const cloneDeep = require('lodash/cloneDeep');
+const isEmpty = require('lodash/isEmpty');
+const isString = require('lodash/isString');
 const { printMessage, formatMessage } = require('formatted-messages');
 
+const isEmptyString = s => isString(s) && isEmpty(s);
+
 const Parser = require('../parser');
-const { CONFIG_FILE_PATH, SCRIPTS_DIRECTORY_PATH, DEFAULT_SHELL } = require('../constants');
-const { throwError, deleteYamlFile } = require('../utility');
+const {
+  ScriptTypes,
+  CONFIG_FILE_PATH,
+  SCRIPTS_DIRECTORY_PATH,
+  DEFAULT_SHELL,
+} = require('../constants');
+const {
+  throwError,
+  deleteYamlFile,
+  isSimpleScript,
+  isValidYamlFileName,
+  isValidJsonFileName,
+  deleteJsonFile,
+} = require('../utility');
 
 const messages = require('./messages');
 
-const getConfig = () => JSON.parse(fse.readFileSync(CONFIG_FILE_PATH, 'utf8'));
+/**
+ * Load the config from the config.json file into memory
+ *
+ * @returns {object} config - config parsed from config.json file
+ */
+const loadConfigJson = () => fse.readJsonSync(CONFIG_FILE_PATH);
+
+/**
+ * Save config to the config.json file
+ *
+ * @param {object} param                - object parameter
+ * @param {string} param.configFilePath - config file path to save to the config.json
+ * @param {string} param.shell          - shell to save to the config.json
+ * @param {string[]} param.scriptNames  - script names to save to the config.json
+ */
+const saveConfigJson = ({ configFilePath, shell, scriptNames }) => {
+  fse.outputJsonSync(
+    configFilePath,
```

Comment:
nit:
- [ ] This can just be `CONFIG_FILE_PATH`. No need to pass it as an argument to this function.